### PR TITLE
Feat/hover styles all icons 3

### PR DESCRIPTION
**Pull request by stranzhay**
repeat times 3. updating hover styles to have cursor pointer. 

1. commitlint is not running from cli
2.  github commitlint is running on reverted commits that are not rebased

### CODE REVIEW

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://615b073158bc281d6bcbe9d4--condescending-leavitt-66e607.netlify.app

File: packages/icon/src/Icon.js [hunk 1]
```diff
@@ -2,11 +2,20 @@ import React from 'react';
...
   <i
     ref={ref}
     aria-hidden="true"
-    className={classNames('icon', `icon-${name}`, size && `icon-${size}`, color && `text-${color}`, className)}
+    className={classNames(
+      'icon',
+      `icon-${name}`,
+      size && `icon-${size}`,
+      color && `text-${color}`,
+      onClick && { cursor: 'pointer' },
```
Comment (jordan-a-young) [inline packages/icon/src/Icon.js]: You can get rid of this line since we have it in the styles

Comment (stranzhay) [reply packages/icon/src/Icon.js]: copy paste. removing now

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://615b0a68cfae7719f93437ef--condescending-leavitt-66e607.netlify.app

**Review by jordan-a-young — APPROVED** [review]

**Pull request merged**